iOSConsole/iOSConsole/run.c

```diff
@@ -88,32 +89,64 @@ void RunAppOnDeviceWithIdentifier(char *udid, char *identifier)
 							CFSafeRelease(setLaunchArgsResponse);
 							SDMMD_DebuggingCommandRelease(setLaunchArgs);
 
-							// setting thread to attach
-							CFMutableArrayRef setThreadArgs = CFArrayCreateMutable(kCFAllocatorDefault, 0x0, &kCFTypeArrayCallBacks);
-							CFArrayAppendValue(setThreadArgs, CFSTR(""));
-							DebuggerCommandRef setThread = SDMMD_CreateDebuggingCommand(kDebugCUSTOMCOMMAND, CFSTR("Hc0"), setThreadArgs);
-							CFSafeRelease(setThreadArgs);
-
-							CFDataRef setThreadResponse = NULL;
-							result = SDMMD_DebuggingSend(dconn, setThread, &setThreadResponse);
-							CFSafeRelease(setThreadResponse);
-							SDMMD_DebuggingCommandRelease(setThread);
-
-							// setting continue with execution
-							CFMutableArrayRef contArgs = CFArrayCreateMutable(kCFAllocatorDefault, 0x0, &kCFTypeArrayCallBacks);
-							CFArrayAppendValue(contArgs, CFSTR(""));
-							DebuggerCommandRef cont = SDMMD_CreateDebuggingCommand(kDebugc, NULL, contArgs);
-							CFSafeRelease(contArgs);
-
-							CFDataRef contResponse = NULL;
-							result = SDMMD_DebuggingSend(dconn, cont, &contResponse);
-							CFSafeRelease(contResponse);
-							SDMMD_DebuggingCommandRelease(cont);
-
-							launchSuccess = true;
-						}
-						if (launchSuccess) {
-							CFRunLoopRun();
+							// Not sleeping for a bit seems to cause debugserver to crash.
+							// https://github.com/libimobiledevice/libimobiledevice/issues/179
+							sleep(1);
```

Comment:
Can you confirm if this sleep is necessary with my code? I ask because if this does cause a crash then I am deviating from how apple's code works in this regard and that is something I would like to investigate.